### PR TITLE
Fixes #15636: Errors with Rudder agent on unprivileged containers (LXC)

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/15636

### CODE REVIEW

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
@amousset changed to check if file exist before forcing the path

**Comment by amousset** [comment]
Can you open the upstream PR too?

**Comment by amousset** [comment]
And could you retarget to 7.2?

File: rudder-agent/SOURCES/patches/cfengine/15636-ignore-symlink-check-on-lxc.patch [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+ 
+     Buffer *pbuf = BufferNew();
+ 
+     JsonElement *inet = JsonObjectCreate(2);
+ 
+-    BufferPrintf(pbuf, "%s/proc/net/netstat", procdir_root);
++    BufferPrintf(pbuf, "%s/proc/%d/net/netstat", procdir_root, promiser_pid);
++    if (!FileCanOpen(BufferData(pbuf), "r"))
++    {
++        BufferPrintf(pbuf, "%s/proc/net/netstat", procdir_root);
```
Comment (ncharles) [inline rudder-agent/SOURCES/patches/cfengine/15636-ignore-symlink-check-on-lxc.patch:17]: shouldn't you change include the promiser_pid in this out ?

Comment (peckpeck) [reply rudder-agent/SOURCES/patches/cfengine/15636-ignore-symlink-check-on-lxc.patch:17]: ???

**Comment by peckpeck** [comment]
PR replaced by https://github.com/Normation/rudder-packages/pull/2727

**Pull request closed without merging**